docs/learn/curvature-drive.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title: "Curvature Drive"
+parent: "Learning"
+permalink: /docs/learn/curvature-drive
+authors: ['hyperliskdev']
+---
+
+# Curvature Drive
+
+Curvature drive is a method inside of the DifferentialDrive class in WPILib 
+
+## What does Curvature Drive do?
+
+Curvature drive is a type of drive just like Arcade or Tank drive. Curvature drive is different in a few key ways;
+
+- isQuickTurn method is used to quickturn on the stop, as it is default off.
+- Turning speed withput isQuickTurn == true, is based on how fast the robot is moving.
+- The robot can be driven with either Trigger Control or Joystick Control
+- And finally, the robot now drives like a car.
+
+## How to make Curvature drive
+
+```java
+
+// Inside DriveTrain.java (hyperliskdev/HyperBot)
```

Comment:
Dont link to your own projects.. This should be codebase-independant. Also, who cares what repo its from?

Reply:
Sorry, referencing things is something I thought was important. That was just a citation.